CHANGELOG.md

```diff
@@ -6,9 +6,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Support for using `Deferred` secrets fetching via `conjur::secret`.
+  [cyberark/conjur-puppet#13](https://github.com/cyberark/conjur-puppet/issues/13).
+
+### Changed
+- `conjur::secret` now must be used as a `Deferred` function. Method signature has
+  changed as well. [cyberark/conjur-puppet#13](https://github.com/cyberark/conjur-puppet/issues/13).
+
 ### Removed
 - Support for using the Conjur Puppet module with Conjur Enterprise v4 is removed
-  [cyberark/conjur-puppet#66](https://github.com/cyberark/conjur-puppet/issues/20).
+  [cyberark/conjur-puppet#66](https://github.com/cyberark/conjur-puppet/issues/66).
+- Support for using this module with Puppet v5.
+  [cyberark/conjur-puppet#104](https://github.com/cyberark/conjur-puppet/issues/104).
+- Support for using host factory tokens, `conjur` class, `cert_file` parameter, and using
```

Comment:
I guess this also means automated population of machine identity is gone.

Reply:
Correct (for now). This is something that we will need to figure out if it's desired or not. It probably is but it's not mandatory for initial release I think.

Reply:
there might be other ways to achieve that down the line by integrating this with our existing authenticators - but it's out of scope for this work, for sure.